src/containers/LivingWhitepaper/LivingWhitepaper.scss

```diff
@@ -110,7 +118,17 @@
 
   &__topics {
     background-color: var(--color-cyan-900);
-    padding: 48px 72px;
+    padding: 72px 48px;
+    padding-right: 72px;
+
+    @media (max-width: 768px) {
+      padding-right: 48px;
+    }
+
+    @media (max-width: 768px) {
```

Comment:
hmm seems like this media query is the same as the one above? should it be `414px` or smth?

Reply:
We have extra padding on the RHS with this change

<img width="606" alt="Screen Shot 2021-07-17 at 9 17 28 AM" src="https://user-images.githubusercontent.com/32864116/126021114-b5db4002-564b-4425-a7c8-8170745583c3.png">

Reply:
fixing that query.

Also, there's actually more padding on RHS. I've left a comment on Figma for you.
But on smaller screens, I have kept it equal because it didn't look nice.